fifo/rtl/internal/BUILD.bazel

```
@@ -94,12 +95,13 @@ verilog_library(
 )
 
 br_verilog_elab_and_lint_test_suite(
-    name = "br_fifo_pop_ctrl_test_suite",
+    name = "br_fifo_pop_ctrl_zero_lat_test_suite",
```

```suggestion
    name = "br_fifo_pop_ctrl_test_suite_zero_lat",
```

(easier to grep for test suite variations for the same module)